packages/model-viewer/src/features/scene-graph.ts

```diff
@@ -149,8 +149,10 @@ export const SceneGraphMixin = <T extends Constructor<ModelViewerElementBase>>(
           return;
         }
 
-        await this[$model]![$switchVariant](variantName!);
+        await this[$model]![$switchVariant](
+            variantName == 'null' ? null : variantName!);
```

Comment:
Did you see  my whole comment on this line?

Reply:
Updated that comment.